### DISCUSSION TITLE
Sorting incoming emails based on receipient

### DISCUSSION THREAD

**Discussion by CoachDom** [the-djmaze/snappymail#1291] [Q&A]
Hi there,
I have multiple emails (contact@domain.tld, work@domain.tld, services@domain.tld and a couple more. 
On tutanota I had it setup that if a mail came to Services, it would automatically go to Services folder.
Is it possible to do here?
Many thanks
Dom

**Comment by cm-schl** [comment]
Hi,

if the "Service folder" is a subfolder of services@domain.tld this could be done with filters accessible in your settings (little gear symbol).
